src/libraries/System.Private.CoreLib/src/System/Collections/ObjectModel/ReadOnlyCollection.cs

```diff
@@ -23,6 +25,25 @@ public ReadOnlyCollection(IList<T> list)
             this.list = list;
         }
 
+        /// <summary>Lazily-initialized empty singleton.</summary>
```

Comment:
There is no other static in this type. You can just depend on the beforefieldinit static constructor to initialize this lazily.

Reply:
ok